modules/core/src/lib/deck.js

```diff
@@ -434,15 +388,19 @@ export default class Deck {
 
   // Callbacks
 
-  _onViewStateChange({viewState}, ...args) {
+  _onViewStateChange(event) {
```

Comment:
Not sure `event` is the best choice of name as it brings browser events to mind. Just call it `parameters` or `opts` maybe?